rmgpy/molecule/element.py

```diff
@@ -71,7 +74,7 @@ class Element:
 
     def __init__(self, number, symbol, name, mass, isotope=-1, chemkinName=None):
         self.number = number
-        self.symbol = intern(symbol)
```

Comment:
what is (was) `intern`?

Reply:
It was a way to register a string to a table to get a slight performance gain because it enabled memory address comparison instead of normal string comparison. However, both of the strings you're comparing must be interned to actually benefit.

In Python 3, `intern` was relocated to `sys.intern`. I figured that we weren't using it properly to benefit, so I decided to remove it completely instead of replacing it.